### PR TITLE
wizard can only roll with 35 players or higher instead of 20

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Tipy1802** [comment]
Please no, traitor only on lowpop is lame

**Comment by ghost** [comment]
@Tipy1802 addressed in recent pr

**Comment by JamieD1** [comment]
Make it 27 for now, we have a wizard change in the works. @AdamElTablawy

File: code/game/gamemodes/wizard/wizard.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	report_type = "wizard"
 	antag_flag = ROLE_WIZARD
 	false_report_weight = 10
-	required_players = 20
+	required_players = 35
```
Comment (JamieD1) [inline code/game/gamemodes/wizard/wizard.dm]:
```suggestion
	required_players = 27
```

**Comment by Reed0506** [comment]
isnt ops 30pop minimum?

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**